actions/DisplayAction.php

```diff
@@ -188,19 +178,48 @@ public function execute()
                         );
 
                         $item->setTimestamp(time());
-                        $item->setContent(buildBridgeException($e, $bridge));
+
+                        $message = sprintf(
+                            'Uncaught Exception %s: %s at %s line %s',
+                            get_class($e),
+                            $e->getMessage(),
+                            trim_path_prefix($e->getFile()),
+                            $e->getLine()
+                        );
+                        $stacktrace = create_sane_stacktrace($e);
+                        $issueBody = sprintf(
+                            "```\n%s\n\n%s\n\nQuery string:%s\nVersion:%s\n```",
+                            $message,
+                            implode("\n", $stacktrace),
+                            $_SERVER['QUERY_STRING'] ?? '',
+                            Configuration::getVersion(),
+                        );
+                        $issueUrl = sprintf('https://github.com/RSS-Bridge/rss-bridge/issues/new?%s', http_build_query([
+                            'title' => sprintf('%s failed with error %s', $bridge->getName(), $e->getCode()),
+                            'body' => $issueBody,
+                            'labels' => 'Bridge-Broken',
+                            'assignee' => $bridge->getMaintainer(),
+                        ]));
+                        $searchUrl = sprintf(
+                            'https://github.com/RSS-Bridge/rss-bridge/issues?q=%s',
+                            urlencode('is:issue is:open ' . $bridge::NAME)
+                        );
```

Comment:
I think this overextends this function bit too much already, centralizing too many facts in a single place, especially with magic constants like the URLs. I would keep the URI construction in separate functions:

```php
                        $issueTitle = sprintf('%s failed with error %s', $bridge->getName(), $e->getCode())
                        $issueUrl = buildGitHubIssueQuery(
                            $issueTitle,
                            $issueBody,
                            'Bridge-Broken'
                            $bridge->getMaintainer(),
                        );
```

They can even be moved into this file (though I think we will probably eventually want to have this for other actions as well).

Reply:
I prefer inlining functions when they are used a single time. When we need github urls other places we can extract it.

Reply:
Well I don't have strong preference here. I added a commit which extracts them.

Reply:
If I am reading right the argument is that it brings:

1. more streamlined stepping through the code using debugger, which is working around inadequate debugger UX
2. preventing accidentally not running state update routine in a nested conditional
3. preventing hidden nested conditionals from introducing performance variability across loop iterations

The first two are only relevant to highly stateful code but meaningless for pure functions like these. The third one applies to pure functions too but I do not think that it applies here either here. So I would say the benefits of encapsulation clearly win in this case.